charts/pg-db/templates/cluster.yaml

```diff
@@ -365,6 +365,9 @@ spec:
     postgresParams: {{ .Values.pmm.postgresParams}}
 
   backups:
+{{ if (hasKey .Values.backups "enabled") }}
```

Comment:
To avoid empty lines on the template try:
 {{ - if

Reply:
Also, all other .Values.backups (like image ...) remains required, probably the end of this if will be after entire backups section